2021/Robot2021/src/main/cpp/commands/DriveShift.cpp

```diff
@@ -43,7 +46,7 @@ bool DriveShift::IsFinished()
 // Called once after isFinished returns true
 void DriveShift::End(bool interrupted)
 {
-    std::printf("2135: DriveShift - End (%d)\n", !m_gearSetting);
+    spdlog::info("DriveShift - End {}", !m_gearSetting);

```

Comment:
Why are we printing !m_gearSetting instead of m_gearSetting? That seems a little bit misleading

Reply:
I'm not sure. I was actually going to ask what the ! does

Reply:
Let's add an item to the TODO list and then we can merge this as is